content/en/docs/v3.5/tutorials/how-to-conduct-elections.md

```diff
@@ -1,16 +1,24 @@
----
-title: How to conduct leader election in etcd cluster
-description: Guide to conducting leader election in an etcd cluster
-weight: 900
----
+## Prerequisites
+
+- Ensure [`etcd`](https://etcd.io/docs/v3.5/install/) and [`etcdctl`](https://etcd.io/docs/v3.5/install/) is installed.
+- Check for active etcd cluster.
 
 `elect` for leader election:
 
-![09_etcdctl_elect_2016050501](https://storage.googleapis.com/etcd/demo/09_etcdctl_elect_2016050501.gif)
+1. The `etcdctl` command is used to conduct leader elections in an etcd cluster. It makes sure that only one client become leader at a time.
+2. Ensure the `ENDPOINTS` variable is set with the addresses of each etcd cluster members.
+3. Set a unique name for the election for different clients ('<i>one</i>' in the given code below).
```

Comment:
Is there a reason for using html tags here for italics rather than standard markdown?

Reply:
> Is there a reason for using html tags here for italics rather than standard markdown?

There wasn't any specific reason, I just wanted to emphasize the character for the reader's comfort. Should I change it??

Reply:
You can certainly add emphasis, but please do it using standard markdown, refer https://www.markdownguide.org/basic-syntax.